### PR TITLE
RHBPMS-4704 - fix to ask for view to resize after full screen mode

### PR DESCRIPTION
**Pull request by tsurdilo**
@jlindop this fixes the issue you saw sometimes where the designer frame does not resize after full screen is enabled.

### CODE REVIEW

File: jbpm-designer-client/src/main/java/org/jbpm/designer/client/DesignerViewImpl.java [hunk 1]
```diff
@@ -34,6 +34,7 @@ public DesignerViewImpl() {
     public DesignerViewImpl(DesignerWidgetView view) {
         this.designerWidget = new DesignerWidgetPresenter(view);
         initWidget(designerWidget.getView().asWidget());
+        this.publisCallOnReisize(this);
```
Comment (jomarko) [inline jbpm-designer-client/src/main/java/org/jbpm/designer/client/DesignerViewImpl.java]: Hi @tsurdilo , I am not sure, but it seems to me as typo. Did you intended name `publicCallOnReisize` ?

**Comment by tsurdilo** [comment]
@jomarko nope was typo :) amended. thanks.

**Comment by jlindop** [comment]
@tsurdilo The build error is due to the issue fixed in https://github.com/kiegroup/jbpm-designer/pull/579

**Comment by tsurdilo** [comment]
jenkins please retest this

**Comment by tsurdilo** [comment]
@jlindop ready to review

**Pull request merged**